src/models/Move.php

```diff
@@ -110,10 +109,24 @@ public static function prepareDescr(?string $descr): array|string|null
         if (empty($descr)) {
             return null;
         }
+        if (str_contains($descr, '//hm4')) {
+            return preg_replace_callback('@https://\S+/(\d+)/?(#\S+)?@', static function ($m) {
+                return Html::a('HM4::' . Html::encode($m[1]), Html::encode($m[0]));
+            }, $descr);
+        } else if (str_contains($descr, 'http') && preg_match('/\/([A-Z]+\-\d+)/', $descr)) {
+            $urlPattern = '/(http|https)\:\/\/?[a-zA-Z0-9\.\/\?]+\/([A-Z]+\-\d+)[a-zA-Z0-9\&\.\/\?\:@\-_=#]*/';
```

Comment:
This regexp takes the issue ID only after the first `/`, so the following URL will not be matched: `https://youtrack.acme.com/agiles/117-99/current?issue=HP-1461`

Search for this pattern ANYWHERE in the Url